packages/rpc-graphql/src/schema/account.ts

```diff
@@ -167,7 +188,24 @@ export const accountTypeDefs = /* GraphQL */ `
 
 export const accountResolvers = {
     Account: {
-        __resolveType(account: { encoding: string; programName: string; accountType: string }) {
+        __resolveType(account: {
+            address: string;
+            encoding: string;
+            executable?: boolean;
+            lamports?: bigint;
+            owner: string;
+            programName: string;
+            accountType: string;
+        }) {
+            if (
+                account.owner === 'Sysvar1111111111111111111111111111111111111' ||
+                account.address === 'Sysvar1nstructions1111111111111111111111111'
+            ) {
+                return 'Sysvar';
+            }
+            if (account.owner === 'NativeLoader1111111111111111111111111111111' || account.executable) {
+                return 'ProgramAccount';
+            }
```

Comment:
It isn't. I meant for it to be a `GenericAccount` because it appears to be that way [in the Explorer](https://explorer.solana.com/address/Feature111111111111111111111111111111111111).